setup.cfg

```diff
@@ -27,4 +27,4 @@ max-line-length=119
 [pydocstyle]
 # Ignore following rules to allow Google Python Style docstrings
 add_ignore = D407,D413
-match_dir = (?!migrations).*
+match_dir = (?!migrations|node_modules).*
```

Comment:
Added this because you were having issues of `node_modules` being checked for docstrings (or something). So I've added this to avoid checking that folder (though it shouldn't be needed).